playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -238,29 +238,22 @@
   any_errors_fatal: true
 
   pre_tasks:
+  - name: Load lib_openshift modules
+    include_role:
+      name: lib_openshift
```

Comment:
I agree on not liking the include_role usage here (and elsewhere).  Depending on the amount of work for implementation, it may be something to come back to (tracked on a card) than work on for this PR.